src/utils/vacation-days-utils.tsx

```diff
@@ -44,28 +51,32 @@ export const renderVacationDaysTextForCard = (person: Person) => {
 /**
  * Display persons vacation days
  *
- * @param Person timebank person
+ * @param Person timebank user
  */
-export const renderVacationDaysTextForScreen = (person: User) => {
-  const spentVacationsColor =
-    person && person.spentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
+export const renderVacationDaysTextForScreen = (user: User) => {
+  // const spentVacationsColor =
+  //   user && user.spentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
+  //
+  // const unspentVacationsColor =
+  //   user && user.unspentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
 
-  const unspentVacationsColor =
-    person && person.unspentVacations > 0 ? theme.palette.success.main : theme.palette.error.main;
+  //FIXME: Deal with the spent and unspent vacations
+  const spentVacationsColor = theme.palette.error.main;
+  const unspentVacationsColor = theme.palette.error.main;
```

Comment:
Seems like we had some repetition here with the colors, perhaps (if we still want to use these colors) we could put these into a function e.g. getVacationColors which can be used in multiple places